core/src/main/resources/org/elasticsearch/tasks/task-results-index-template.json

```diff
@@ -0,0 +1,52 @@
+{
+  "template" : ".results",
+  "order" : 2147483647,
+  "settings" : {
+    "number_of_shards" : 1,
+    "number_of_replicas" : 1,
+    "index.priority" : 2147483647
```

Comment:
This seems like a really high priority! I guess it is ok because it should be small?